apps/syntrest/src/main/resources/application.properties

```diff
@@ -4,7 +4,9 @@ synth-medl-url=https://nais-synthdata-medl.nais.preprod.local/api/v1/generate_me
 synth-inst-url=https://nais-synthdata-inst.nais.preprod.local/api/v1/generate_inst/{numToGenerate}
 synth-arena-meldekort-url=https://nais-synthdata-arena-meldekort.nais.preprod.local/api/v1/generate_meldekort/{numToGenerate}/{meldegruppe}
 synth-arena-bisys-url=https://nais-synthdata-arena-bisys.nais.preprod.local/api/v1/generate_bisys/{numToGenerate}
-synth-inntekt-url=https://nais-synthdata-inntekt.nais.preprod.local/api/v1/generate
+synth-inntekt-url=https://nais-synthdata-inntekt.nais.preprod.local/api/v1/generate/inntekt
+synth-arena-aap-115-url=https://nais-synthdata-arena-aap.nais.preprod.local/api/v1/generate_arena_aap/11_5/{numToGenerate}
```

Comment:
All støtte for arena er fjernet fra syntrest siden de rest-endepunktene ikke fungerte lenger, så `synth-arena-aap-115-url` og `synth-arena-aap-nyRettighet-url` skal være fjernet.

Reply:
Fjerner dem og merger PR-en ;)